packages/salesforcedx-vscode-core/package.json

```diff
@@ -989,6 +989,11 @@
           "type": "boolean",
           "default": false,
           "description": "%enable_sobject_refresh_on_startup_description%"
+        },
+        "salesforcedx-vscode-core.activate-source-tracking-for-deploy-retrieve": {
```

Comment:
total nit: Mostly curious.  I noticed that the detectConflictsAtSync setting looks nicer in the settings panel.  Looking at https://github.com/microsoft/vscode-extension-samples/blob/4e9b69f212d4544f8ad933b473c84db87b34c78f/configuration-sample/package.json it appears the camel case is also how they roll over in microsoft land.  I wonder if we should make the switch for these configuration values.  I did initially think the setting was broken due to the kebab-case

Reply:
Yeah, agree it looks nicer and good to see how they are doing it.  Maybe we can update this one to use camel case to get the formatting in the UI, and create a follow up ticket to track the work to change the others?

@sbudhirajadoc What do you think?  Camel case in these config values makes the text look more properly formatted in the settings UI:
<img width="1266" alt="2022-04-06_10-57-53" src="https://user-images.githubusercontent.com/46458081/162039021-5822bce4-4cb2-43bb-9692-639dec971e86.png">

Reply:
yeah...def feels very scope creepy to expand what you've already done that matches most everything else :).  Let's just capture this in a new ticket and we can address on the whole. Do you want to create a ticket or should I?

Reply:
I'll create it!

Reply:
Ticket to track for follow up: https://github.com/forcedotcom/easywriter/issues/20

Reply:
Please say--
 **Activate Deploy and Retrieve for Source Tracked Orgs**
Specifies whether to enable the Org Browser and allow use of deploy and retrieve commands when connected to a source-tracked org.